### DISCUSSION TITLE
Add integration to google calendar

### DISCUSSION THREAD

**Discussion by camagenta** [novalagung/muslimboard#72] [Ideas]
Akh Agung, Jazakumullahu khoiron for developing this plugin. 

btw, i don't really todolist on this plugin because it not sync with another platform. then after sometime, I think after awhile it would very usefull if we can see our agenda on google calendar on muslimboard screen. the calendar displayed on agenda mode so we remember our next meeting based on google calendar.

I am happy to hear what you think about this idea

Barakallahu fiikum

**Comment by novalagung** [comment]
@camagenta  Hi mas Kurnia, long time no see.

I do agree that I need to do something with the TODO list. Synchronizing it with Google Calendar or another platform would be nice. 

The current version of Muslim Board fully operates on the browser side, adding new integration would require some backend setup, but that's completely fine.

Let me try to do some research on my end, and I shall go back to this discussion. I appreciate your suggestion, thanks!
